src/SprykerEco/Zed/PunchoutCatalogs/Business/Reader/PunchoutCatalogsReader.php

```diff
@@ -65,4 +67,15 @@ public function findConnectionByIdWithPassword(int $idConnection): ?PunchoutCata
 
         return $punchoutCatalogConnectionTransfer->setPassword($password);
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\PunchoutCatalogConnectionFilterTransfer $punchoutCatalogConnectionFilterTransfer
+     *
+     * @return bool
+     */
+    public function hasPunchoutCatalogConnection(
+        PunchoutCatalogConnectionFilterTransfer $punchoutCatalogConnectionFilterTransfer
+    ): bool {
+        return $this->punchoutCatalogsRepository->hasPunchoutCatalogConnection($punchoutCatalogConnectionFilterTransfer);
```

Comment:
1-line method, can we use it directly?

Reply:
Khm, sure I can, but as for me its more clearly to use it in this way.